htdocs/js/modules/direct-entry-react.js

```diff
@@ -52,16 +52,23 @@ window.onload = function() {
   const context = JSON.parse(instrumentEl.dataset.context);
   const initialData = JSON.parse(instrumentEl.dataset.initial);
   const lang = instrumentEl.dataset.lang;
+  const logo = instrumentEl.dataset.logo ? instrumentEl.dataset.logo : "";
   const options = { surveyMode: true };
-
+  const curDate = new Date();
+  const curDateSt = (curDate.getMonth()+1) + "/" + curDate.getDate() + "/" + curDate.getFullYear();
   ReactDOM.render(
-    <DirectEntryReact
-      instrument={instrument}
-      initialData={initialData}
-      lang={lang}
-      context={context}
-      options={options}
-    />,
+    <div>
+      <img id="banner" className="banner"/>
+      <img id="logo" src={logo}/>
+      <div><font color="white">Date: {curDateSt}</font></div>
```

Comment:
Can you put this stuff, (the parent div and banner stuff) into DirectEntryReact#render ?